pkg/controller/clusterconfig/migration.go

```diff
@@ -103,7 +103,7 @@ func (r *ReconcileClusterConfig) prepareOperatorConfigForNetworkTypeMigration(ct
 		return nil
 	}
 
-	if !cniReady && clusterConfig.Status.Migration == nil {
+	if !cniReady && (clusterConfig.Status.Migration == nil || clusterConfig.Status.Migration.NetworkType == "") {
```

Comment:
Is `clusterConfig.Status.Migration.NetworkType == ""` still possible with https://github.com/openshift/cluster-network-operator/pull/2651/files#diff-0490e0f88e14c4f85aff51f6745b80b3bdfb7867b5b6bbb890945cac20cec481R362?

Reply:
No, it's solely for safety.